[ERROR] Unhandled error! with -b

Hello,

since the last version (3.4.3) the option “-b” to pull the dependencies from the FreeBSD repository seems to bring poudriere into panic.

Below an snipped of the log (already with -vv)

Reverting back to 3.4.2 seems to workaround the problem at the moment that i needed that option.

```
[00:00:09] Trimming IGNORED and blacklisted ports
[00:00:09] Package fetch: Looking for missing packages to fetch from pkg+http://pkg.FreeBSD.org/${ABI}/latest
[00:00:09] Packge fetch: bootstrapping pkg
Bootstrapping pkg from pkg+http://pkg.FreeBSD.org/FreeBSD:14:amd64/latest, please wait...
Verifying signature with trusted certificate pkg.freebsd.org.2013102301... done
[143Samd64-default] Installing pkg-2.2.1...
[143Samd64-default] Extracting pkg-2.2.1: 100%
Updating FreeBSD repository catalogue...
[143Samd64-default] Fetching meta.conf: 100%    179 B   0.2kB/s    00:01    
[143Samd64-default] Fetching data.pkg: 100%   10 MiB   2.7MB/s    00:04    
Processing entries: 100%
FreeBSD repository update completed. 36462 packages processed.
FreeBSD is up to date.
[ERROR] Unhandled error!
[00:00:24] Cleaning up
[00:00:24] Unmounting file systems
Exiting with status 1
```

Thank you in advance.

--- a/src/share/poudriere/common.sh
+++ b/src/share/poudriere/common.sh
@@ -2776,11 +2776,26 @@ enter_interactive() {
 	msg "Installing local Pkg repository to ${LOCALBASE}/etc/pkg/repos"
 	mkdir -p ${MASTERMNT:?}${LOCALBASE:?}/etc/pkg/repos
 	cat > ${MASTERMNT:?}${LOCALBASE:?}/etc/pkg/repos/local.conf <<-EOF
-	FreeBSD: { enabled: no }
-	FreeBSD-kmods: { enabled: no }
-	FreeBSD-ports: { enabled: no }
-	FreeBSD-ports-kmods: { enabled: no }
-	FreeBSD-base: { enabled: no }
+	FreeBSD: {
+		enabled: no,
+		priority: 100
+	}
+	FreeBSD-kmods: {
+		enabled: no,
+		priority: 100
+	}
+	FreeBSD-ports: {
+		enabled: no,
+		priority: 100
+	}
+	FreeBSD-ports-kmods: {
+		enabled: no,
+		priority: 100
+	}
+	FreeBSD-base: {
+		enabled: no,
+		priority: 100
+	}
 
 	local: {
 		url: "file:///packages",
@@ -4688,15 +4703,30 @@ download_from_repo() {
 		pkg_bin="pkg"
 	fi
 	cat >> "${MASTERMNT:?}/etc/pkg/poudriere.conf" <<-EOF
-	FreeBSD: { enabled: no }
-	FreeBSD-kmods: { enabled: no }
-	FreeBSD-ports: { enabled: no }
-	FreeBSD-ports-kmods: { enabled: no }
-	FreeBSD-base: { enabled: no }
+	FreeBSD: {
+		enabled: no,
+		priority: 100
+	}
+	FreeBSD-kmods: {
+		enabled: no,
+		priority: 100
+	}
+	FreeBSD-ports: {
+		enabled: no,
+		priority: 100
+	}
+	FreeBSD-ports-kmods: {
+		enabled: no,
+		priority: 100
+	}
+	FreeBSD-base: {
+		enabled: no,
+		priority: 100
+	}
 
 	Poudriere: {
-	        url: ${packagesite};
-	        mirror_type: $(if [ "${packagesite#pkg+}" = "${packagesite}" ]; then echo "none"; else echo "srv"; fi);
+		url: ${packagesite},
+		mirror_type: $(if [ "${packagesite#pkg+}" = "${packagesite}" ]; then echo "none"; else echo "srv"; fi)
 	}
 	EOF
 

--- a/src/share/poudriere/jail.sh
+++ b/src/share/poudriere/jail.sh
@@ -926,11 +926,26 @@ pkgbase: {
 }
 EOF
 	cat <<EOF > "${JAILMNT}/etc/pkg/FreeBSD2.conf"
-FreeBSD: { enabled: no }
-FreeBSD-kmods: { enabled: no }
-FreeBSD-ports: { enabled: no }
-FreeBSD-ports-kmods: { enabled: no }
-FreeBSD-base: { enabled: no }
+FreeBSD: {
+	enabled: no,
+	priority: 100
+}
+FreeBSD-kmods: {
+	enabled: no,
+	priority: 100
+}
+FreeBSD-ports: {
+	enabled: no,
+	priority: 100
+}
+FreeBSD-ports-kmods: {
+	enabled: no,
+	priority: 100
+}
+FreeBSD-base: {
+	enabled: no,
+	priority: 100
+}
 EOF
 
 	pkg -o IGNORE_OSVERSION=yes -o REPOS_DIR="${JAILMNT}/etc/pkg" -o ABI="FreeBSD:${VERSION}:${ARCH}" -r ${JAILMNT}/ update

